### ISSUE TITLE
SESSION.global()

### ISSUE THREAD

**Issue by CrystalMoogle** [po-devs/pokemon-online#881]
Since the last server update, it seems as if SESSION.global() clears itself after a script update. I can't really test properly, since server built from source crashes when I join, but things that were relying on it being unchanged beforehand, suddenly don't work.
One that's fairly easy to spot is /uptime, trying to use it on the main server gives an error. Trivia and battlehistory both rely on it as well, but they're less obvious.
SESSION.global() was used because it used to keep all the data for the server session without ever clearing it, unless someone told it to. Now that it clears on script update, it's kinda annoying, since it would mean a change in how things are done and I don't really think scripts should be changed for something that was working fine beforehand.
Basically, I just want it back to how it was :(
